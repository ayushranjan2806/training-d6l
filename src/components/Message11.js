import React from 'react'

const Message11 = ({message}) => {
  return (
    <div>{message}</div>
  )
}

export default Message11