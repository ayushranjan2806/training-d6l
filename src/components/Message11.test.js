import React from 'react';
import renderer from 'react-test-renderer'


import Message11 from './Message11';

test('Message snapshot with differet',()=>{
    const component = renderer.create(<Message11 message="click the ayish"/>);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot(); 
})