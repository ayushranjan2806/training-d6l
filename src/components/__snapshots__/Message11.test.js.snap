// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Message snapshot with differet 1`] = `
<div>
  click the ayish
</div>
`;
